modules/sfp_threatjammer.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:        sfp_threatjammer
+# Purpose:     Check if an IP or netblock is malicious according to ThreatJammer.com.
+#
+# Author:      diego.parrilla.santamaria@gmail.com
+#
+# Created:     05/05/2022
```

Comment:
International date format is preferred, future man.

```suggestion
# Created:     2022-05-03
```

Reply:
Time-machine fixed